### ISSUE TITLE
No update notification for oC 9.0.2

### ISSUE THREAD

**Issue by ghost** [owncloud/core#24827]
### Steps to reproduce
1. Install oC 9.0.1
2. Open http://example.com/index.php/settings/admin#updater
3. See: "Your version is up to date. Checked on May 25, 2016 at 7:55:08 AM"
### Expected behaviour

An update to oC 9.0.2 should be shown.
### Actual behaviour

No update to oC 9.0.2 is provided. 

The main issue here is that there is yet again another bug in the updater app https://github.com/owncloud/updater/issues/350 . A decision was made in https://github.com/owncloud/updater/issues/350#issuecomment-218815817 that 9.0.2 is not provided to the enduser via the updater app.

This also can be seen at the updater URL like: https://updates.owncloud.com/server/?version=9x0x0x7x1382907615.86x1435439812xstablexx2015-02-07T02:34:55+00:00

To fix this issue at least a update notification should be shown to notify users that there is a new version and that they can update manually to a newer release.

**Comment by nickvergessen** [comment]
Yeah sounds like a good solution, the problem is that both, the updater and the notifications use the same URL. Maybe that is something we should fix at some point.

**Comment by RealRancor** [comment]
Maybe the xml provided by the update server:

```
<?xml version="1.0" encoding="UTF-8"?>
<owncloud>
 <version>8.2.5</version>
 <versionstring>ownCloud 8.2.5</versionstring>
 <url>https://download.owncloud.org/community/owncloud-8.2.5.zip</url>
 <web>https://doc.owncloud.org/server/8.2/admin_manual/maintenance/upgrade.html</web>
</owncloud>
```

could be extended like:

```
<?xml version="1.0" encoding="UTF-8"?>
<owncloud>
 <version>8.2.5</version>
 <versionstring>ownCloud 8.2.5</versionstring>
 <url>https://download.owncloud.org/community/owncloud-8.2.5.zip</url>
 <web>https://doc.owncloud.org/server/8.2/admin_manual/maintenance/upgrade.html</web>
 <updater_enabled>false/true<updater_enabled>
</owncloud>
```

Then the updater could use that to provide an update or not?

**Comment by nickvergessen** [comment]
Well but we still want the updater to update people from 8.2.4 to 8.2.5

**Comment by RealRancor** [comment]
Sure, but the flag false/true still could be variable. So e.g. if:

https://updates.owncloud.com/server/?version=8x2x4x7x1382907615.86x1435439812xstablexx2015-02-07T02:34:55+00:00

is called it has ``<updater_enabled>true<updater_enabled>`` but if:

https://updates.owncloud.com/server/?version=9x0x0x7x1382907615.86x1435439812xstablexx2015-02-07T02:34:55+00:00

is called it has ``<updater_enabled>false<updater_enabled>``

**Comment by RealRancor** [comment]
Additional suggestion:

```
<updater_not_available_reason>The update to oC 9.0.2 is currently not available via the updater app due to a bug in this app.</updater_not_available_reason>
```

**Comment by nickvergessen** [comment]
Okay I misphrased, let me try again.
Let's say 8.2.5 updater is broken, so for 8.2.4 the update is not shipped. But for people with 8.2.3 it would also say 8.2.5 with no updater, although they could first do the update to 8.2.4 with the updater.

But I agree, thats details and it would at least improve some situations, while not all, but babysteps-to-victory

**Comment by RealRancor** [comment]
Yes, but AFAIK the new update server is capable to provide different responses for different passed oC versions to the updater url as described above: https://github.com/owncloud/core/issues/24827?_pjax=%23js-repo-pjax-container#issuecomment-221513656

**Comment by kriztan** [comment]
Are there any news on this issue or fixes without installing update manually?

**Comment by nickvergessen** [comment]
you can download the zip and update manually, that should work just fine.

**Comment by dergilb99** [comment]
> you can download the zip and update manually, that should work just fine.

Sure, but's a pain and takes a lot longer.

**Comment by nightspotlight** [comment]
experiencing same issue here with 9.1.3 when 9.1.4 is available. got to upgrade manually?
upgraded to 9.1.3 before manually as well, so this issue has been there for some time.
upgrading manually takes more time and is less convenient.

**Comment by luckydonald** [comment]
My OC **8.2.7** displayed "Updates **Your version is up to date**".
And today our datacenter learned about https://scan.nextcloud.com/results/
warning that **9.1** is the most current. 

Yeah, saying _You have the most recent version already_ is just misleading.

**Comment by kdslkdsaldsal** [comment]
@luckydonald You can't trust the ownCloud internal update notification. This is a long running issue since years. Instead just check https://owncloud.org/changelog/ from time to time to see the available versions.

**Comment by bcutter** [comment]
I´m pretty sure I´m not at the right point here - doesn´t matter, **point me to it**.

Why the hell are we still stuck on 9.1.2(.5) instead of 9.1.4 or at least 9.1.3 which was released THREE MONTHS AGO?

It´s even not a good sign for itself being forced as a security interested "normal OwnCloud user" to search for reasons on a developer platform. There´s just no communication. No reason. No solution. Very frustrating. See it from the user´s side!

**Comment by PVince81** [comment]
update server progress now tracked here: https://github.com/owncloud/administration, usually as PRs
